src/brokers/utils/add-broker.ts

```diff
@@ -43,7 +77,10 @@ export const newArtemisCRState = (namespace: string): FormState => {
   return {
     shouldShowYAMLMessage: true,
     editorType: EditorType.BROKER,
-    cr: initialCr,
+    cr: initCr,
+    editorActiveProperties: {
+      activeProperties: new Map<string, any>(),
```

Comment:
We can make the type a notch more precise:
```
 activeProperties:{
  itemId: 'acceptors' | 'connectors' | 'console'`;
  expanded: boolean:
}
```

Reply:
should `itemId` be unique for each acceptor/connector as there may be multiple acceptor/connectors
If user expands an acceptor the editor need to know which acceptor/connector is expanded

Reply:
for example I need to remember that **acceptor0** and **acceptor2** are expanded, but **acceptor1** is not
![Screenshot from 2024-06-12 23-18-14](https://github.com/artemiscloud/activemq-artemis-self-provisioning-plugin/assets/686319/72aaf4bb-ff94-491b-a162-3735ebddfc14)

Reply:
I understand now, that makes sense! I thought it was only for the bigger categories. Let's chat about it in today's UX meeting.